circle.yml

```diff
@@ -427,7 +427,7 @@ jobs:
     steps:
       - build
       - download_execution_tests:
-          commit: e279c3d0a53e2f3055c7d7f72f993b62dde18e36
+          commit: v11.3
```

Comment:
The diff between e279c3d0a53e2f3055c7d7f72f993b62dde18e36 only seems to be concerned with EOF-related test changes.

Reply:
I just picked the latest release which was newer than the previous hash.